src/main/java/net/mcreator/blockly/data/ToolboxBlock.java

```diff
@@ -36,6 +36,7 @@
 
 	@Nullable public List<String> fields;
 	@Nullable public List<String> inputs;
+	@Nullable public List<AdvancedInput> advancedInputs;

```

Comment:
Due to JSON naming conventions, fields that are used in JSON are snake case here